tests/e2e/cucumber/no-results.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('no results message is displayed', () => {
+  cy.getByDataTest('no-results').should('exist');
+});
+
+And('there are no results', () => {
+  cy.getByDataTest('total-results').should('not.exist');
```

Comment:
Instead of checking for the `total-results` label, wouldn't it make more sense to actually assert that there are not `search-grid-result` ?